2.py

```diff
@@ -0,0 +1,11 @@
+# Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+# Пример:
+# - [2, 3, 4, 5, 6] => [12, 15, 16];
+# - [2, 3, 5, 6] => [12, 15]
+
+import math
+list = [2, 3, 4, 5, 6]
+resultat = []
+for i in range (math.ceil (len(list)/2)):
+    resultat.append(list [i]*list[-i-1])
+print (resultat)
```

Comment:
```suggestion
for i in range((len(list) + 1) // 2):
    resultat.append(list[i] * list[-i - 1])
print(resultat)
```
Так немного лучше, не правда ли?